include/libavcodec/avcodec.pxd

```diff
@@ -7,6 +7,14 @@ from libc.stdint cimport (
 
 
 cdef extern from "libavcodec/avcodec.h" nogil:
+    """
```

Comment:
This comment still needs to go

Reply:
This is needed to compile on older FFmpeg versions, else `AV_FRAME_DATA_SEI_UNREGISTERED` does not exist. Or is there an other way to deal with this?

Reply:
Oh, wait so I totally misunderstood. You're telling me Cython is treating a docstring as C and compiling it?

Reply:
It does! See [here](https://cython.readthedocs.io/en/latest/src/userguide/external_C_code.html#including-verbatim-c-code).

Reply:
Wow OK I really learned something today! In that case why do we even need the version check in the enum, can't we key off `lib.AV_FRAME_DATA_SEI_UNREGISTERED` being -1?

Reply:
Good idea, that should work indeed. I'll update the PR!